ocf_datapipes/load/__init__.py

```diff
@@ -0,0 +1,14 @@
+# from .configuration import OpenConfigurationIterDataPipe as OpenConfiguration
```

Comment:
do we need to put these in the init? does this mean they will always ben laoded, but mabe we dont need them?

Reply:
Yes, but it gives nicer imports, and I was trying to match torchdata's IterDatPipe code, which does it like this. None of these should have large requirements, the only one that does is the OpenTopography one, which is put in the try/except block below, the rest only require more basic packages. Having them here also makes it simpelr to import than from a ton of the different subfiles, which I think makes it more usable to users as well.